sszgen/main.go

```diff
@@ -28,9 +28,11 @@ func main() {
 	var output string
 	var include string
 	var experimental bool
+	var excludeObjs string
 
 	flag.StringVar(&source, "path", "", "")
 	flag.StringVar(&objsStr, "objs", "", "")
+	flag.StringVar(&excludeObjs, "exclude-objs", "", "Comma-separated list of types to exclude from output")
```

Comment:
This is a new flag to ask fastssz to skip over named structs in codegen. It was helpful when we wrote the Transaction type by hand while prototyping these changes (otherwise fastssz would fail on parse errors before the other tag parsing changes in the PR were introduced). I think it's generally useful but could be excluded from this PR.